ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -135,7 +135,8 @@
     # Note - had to have these set to `install-mock` to progress role
     # installation before changing config to restored DB from other system.
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.3.1
+      omero_server_release: 5.3.2
+      omero_server_upgrade: True
```

Comment:
I'd imagine running with `-C` would be fine, but if that's not green (with the exception of the pip weirdness) then I definitely wouldn't run it against production, which should be our goal with tidying up.